src/index.html

```diff
@@ -17,6 +17,22 @@
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <a href = "#" class="logo">
+        <img src="images/logo.png" alt="Moyo Logo">
+      </a>
+      <nav>
+        <ul>
+          <li><a href="#"class ="is-active">Apple</a></li>
```

Comment:
There should be a space between `href="#"` and `class="is-active"`. This is an HTML syntax issue that could affect rendering.